js/api/modules/todoItem.js

```diff
@@ -0,0 +1,35 @@
+import { api } from "/js/api/index.js";
+import { $loadingBar } from "/js/view/loadingbar.js";
+
+export const todoItemApi = {
+  async toggleItem(member, itemId) {
+    $loadingBar.loading();
+    return await api.put(`/api/users/${member._id}/items/${itemId}/toggle`);
+  },
+
+  async saveItem(member, contents) {
+    $loadingBar.loading();
+    return await api.post(`/api/users/${member._id}/items/`, {
+      contents: contents,
+    });
+  },
+
+  async deleteItem(member, itemId) {
+    $loadingBar.loading();
+    return await api.delete(`/api/users/${member._id}/items/${itemId}`);
+  },
+
+  async editItem(member, itemId, contents) {
+    $loadingBar.loading();
+    return await api.put(`/api/users/${member._id}/items/${itemId}`, {
+      contents: contents,
+    });
+  },
+
+  async setPriority(member, itemId, priority) {
+    $loadingBar.loading();
+    return await api.put(`/api/users/${member._id}/items/${itemId}/priority`, {
+      priority: priority,
+    });
```

Comment:
ES6의 property shorthand(단축 속성명)라는 작성 방법이 있는데요.
key와 value가 같다면 한번만 표기해서 작성할 수 있습니다. 그래서 아래와 같이 간략히 작성하실 수 있어요 ㅎㅎ

```suggestion
return await api.put(`/api/users/${member._id}/items/${itemId}/priority`, { priority });
```